index.html

```diff
@@ -168,31 +168,41 @@ <h1 class="main-text">About Myself</h1>
-                            </li>
-                            <li class="lang">
-                                <a href="#"><img class="lang-logo" src="images/html-logo.png" alt="html"></a>
-                                <p class="upper">Html</p>
-                            </li>
-                            <li class="lang">
-                                <a href="#"><img class="lang-logo" src="images/css-logo.png" alt="css"></a>
-                                <p class="upper">css</p>
-                            </li>
-                        </ul>
-                    </li>
-                    <li><a href="#"><img src="images/frameworks.png" alt="frameworks"></a></li>
-                    <li><a href="#"><img src="images/skills.png" alt="skills"></a></li>
-                </ul>
-            </div>
-        </section>
-    </section> 
-</body>
+        <div class="languages">
+            <ul class="coder">
+                <li><a href="#"><img src="images/languages.png" alt="languages"></a></li>
+                <li>
+                    <ul class="lang-container">
+                        <li class="lang">
+                            <a href="#"><img class="lang-logo" src="images/javasript-logo.png" alt="javascript"></a>
+                            <p class="upper">Javascript</p>
+                        </li>
+                        <li class="lang">
+                            <a href="#"><img class="lang-logo" src="images/html-logo.png" alt="html"></a>
+                            <p class="upper">Html</p>
+                        </li>
+                        <li class="lang">
+                            <a href="#"><img class="lang-logo" src="images/css-logo.png" alt="css"></a>
+                            <p class="upper">css</p>
+                        </li>
+                    </ul>
+                </li>
+                <li><a href="#"><img src="images/frameworks.png" alt="frameworks"></a></li>
+                <li><a href="#"><img src="images/skills.png" alt="skills"></a></li>
+            </ul>
+        </div>
+    </section>
+    <section class="my-form">
+        <div class="my-form-container">
+            <h1 class="text-white text-center">Contact me</h1>
+            <p class="text-center">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+            <form action="https://formspree.io/f/xvodvroe" method="post">
+                <input type="text" id="name" required placeholder="Yeremias" maxlength="30" class="full-width height-input space pad-x-y">
+                <input type="email" id="email" required placeholder="natajayanj14@gmail.com" class="full-width height-input space pad-x-y">
+                <textarea name="message" id="message" required cols="30" rows="10" maxlength="500" placeholder="Write your message here" class="full-width space pad-x-y"></textarea>
```

Comment:
- [ ] Kindly let us give to the `<textarea>` the same `font-family` as in the one in the Figma template, because for now your `<textarea>` is using the default  `font-family`, please let's fix this issue 👍